renderer/src/hooks/StationWallet.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState, useEffect } from 'react'
+import {
+  getDestinationWalletAddress,
+  setDestinationWalletAddress,
+  getStationWalletAddress,
+  getStationWalletBalance,
+  getStationWalletTransactionsHistory
+} from '../lib/station-config'
```

Comment:
can we move the `StationWallet*` functions to a separate lib? They aren't part of Station's config

Reply:
Definitely. I'll create a followup PR for this since it will affect #334 and #335